src/ZkFold/Symbolic/Cardano/Types/Value.hs

```diff
@@ -30,7 +30,7 @@ deriving instance
     , KnownNat (TypeSize context (SingleAsset context))
     ) => SymbolicData context (Value n context)
 
-instance (FromConstant Natural (UInt 64 Auto context), MultiplicativeSemigroup (UInt 64 Auto context))
+instance (Symbolic context, FromConstant Natural (UInt 64 Auto context), MultiplicativeSemigroup (UInt 64 Auto context))
```

Comment:
Maybe with `Symbolic context` constraint, we can get rid of `FromConstant Natural ...` and `MultiplicativeSemigroup ...` constraints?